api/provider/aws/load_balancer_delete.go

```diff
@@ -28,23 +28,24 @@ func (l *LoadBalancerProvider) Delete(loadBalancerID string) error {
 	}
 
 	// Check for eventually consistency
-	waitUntilLBDeletedFN := func() (shouldRetry bool, err error) {
+	var err error
+	waitUntilLBDeletedFN := func() (shouldRetry bool) {
 		loadBalancerName := fqLoadBalancerID
 		if _, err = describeLoadBalancer(l.AWS.ELB, l.AWS.ALB, loadBalancerName); err != nil {
```

Comment:
we should be making an aws calls on these retries to check for eventual consistency